src/components/inline/Button/Button.module.css

```diff
@@ -3,7 +3,7 @@
   border: 1px solid var(--color-brand);
   color: var(--color-white);
   cursor: pointer;
-  font-family: var(--font-family-base);
+  font-family: var(--font-family-body);
```

Comment:
I see the other pull request at https://github.com/ShelterTechSF/sheltertech.org/compare/1648-create-global-variables-colors-and-fonts may give a conflict. So I would change this back and rebase this commit after merging ticket 1648.